.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# Xi-Mac Issue Template
+- [ ] I have searched existing issues and could not find my issue.
+- [ ] I have studied the documentation.
+
+<!---
+Please ensure the issue meets these requirements. If you are not sure, questions
+are welcome on the #xi-mac channel on https://xi.zulipchat.com.
+--->
+## Details
+
+_If your issue is a build or runtime error, please include the following:_
+- macOS version (e.g. macOS 10.13.2)
+- the commit you're on, if building from source (`#3a2405b`)
+- a full backtrace or error message, if available
+
+## Expected vs Actual
```

Comment:
maybe we can delete everything below this line for xi-mac?

Reply:
Amended with that change!